CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.4.31 (Feb 2021)
+  * [KT-44295](https://youtrack.jetbrains.com/issue/KT-44295) Fix Kotlin/Native compiler crash on Android NDK
+  * [KT-44826](https://youtrack.jetbrains.com/issue/KT-44826) Fix "Backend Internal error: Exception during IR lowering" when building MacOS targets
+  * [KT-44764](https://youtrack.jetbrains.com/issue/KT-44764) MPP iOS framework build fails with AssertionError: FUN name:onError_6 visibility:public modality:OPEN
+  * [GH-4588](https://github.com/JetBrains/kotlin-native/pull/4588) Fix bug with calling fake override interface methods
```

Comment:
Might be somewhat unclear to users. "Fix runtime crash in createTypeInfo in release framework binaries" would describe the symptoms more accurately.